doc/src/boundary-initial-conditions/soln-bcs.rst

```diff
@@ -89,6 +89,73 @@ dependent, boundary condition labelled *name* in the .pyfrm file with
         - ``p`` --- static pressure
 
            *float* | *string*
+      
+    ``char-riem-inv-mass-flow`` only works with ``euler`` | ``navier-stokes`` 
+    and requires
+
+        - ``rho`` --- density
+
+           *float* | *string*
+
+        - ``u`` --- x-velocity
+
+           *float* | *string*
+
+        - ``v`` --- y-velocity
+
+           *float* | *string*
+
+        - ``w`` --- z-velocity
+
+           *float* | *string*
+
+        - ``p`` --- initial static pressure, the controller will vary this to
+          target a mass flow rate.
+
+           *float* | *string*
```

Comment:
Can this be a string?

Reply:
Sorry, I thought cfg.getfloat automatically evaluated constants/expressions in the ini file - I now see that is handled differently, I'll correct this.